### ISSUE TITLE
Inputs during training and inference

### ISSUE THREAD

**Issue by zchwang** [tyang816/ProtSSN#5]
Hi,

Excellent work! I am reading ProtSSN and trying to use it, but I have a few questions: 
1) The input in downstream tasks is the entire protein, while the training set uses CATH domains. How does this difference affect the model's performance?
2) The model's inputs during training are crystal structures, but AF2 or ESM2 predicted structures are used for inference. How much bias does this introduce?
3) If I want to use ProtSSN for downstream tasks, do I just use the code provided in README to extract embeddings?

Congratulations again on your work! 

Best regards

**Comment by tyang816** [comment]
Hi, Wang,

1. Good question! From a biological perspective, the CATH domain already contains sufficient protein structure paradigms, but from a computational perspective, this is indeed a gap, and we will conduct additional experimental tests in the future.
2. We don't know how much error this will cause in dry experiments, because we can't get the crystal structure of most proteins. But we are currently doing wet experiment verification, and it seems that both the predicted structure and the crystal structure work well. We may be able to answer this question in our future iterative work.
3. I have added the new code for fine-tuning ProtSSN on any downstream tasks, you can see [here](https://github.com/tyang816/ProtSSN/tree/master?tab=readme-ov-file#-fine-tuning-on-downstream-tasks). You could provide CSV with labels and PDB files for training, the dataset formation can be found [here](https://github.com/tyang816/ProtSSN/tree/master/data/finetune_example/PDBSol).

Thank you for your attention, and welcome to follow our latest work [ProSST](https://github.com/ai4protein/ProSST).😊
